src/endpoints/v1.es6.js

```diff
@@ -903,6 +911,43 @@ class APIv1Endpoint {
     }, this);
   }
 
+  updateCommentOrLink (options) {
+    var uri = options.origin + '/api/editusertext';
+
+    if (!options.model) {
+      throw new NoModelError('/api/editusertext');
+    }
+    // api only supports updating selftext
+    var prop = options.model.get('selftext') ? 'selftext' : 'body';
```

Comment:
It would be safer and more clear to check if `options.model._type === 'Link'` or `'Comment'` instead of checking `selftext`, which could be an empty string.